src/endpoints/nodes/node.service.ts

```diff
@@ -269,6 +271,17 @@ export class NodeService {
     }
   }
 
+  private async applyNodeUnbondingPeriods(nodes: Node[]): Promise<void> {
+    await Promise.all(nodes.map(async node => {
```

Comment:
Promise.all should be made only on the leaving nodes, because only then an await will be performed

Reply:
fixed